app/src/main/java/kr/ac/kaist/team888/locator/Locator.java

```diff
@@ -680,6 +681,29 @@ public void manipulateSkeleton(double flatteningControl, double ariseControl) {
     }
   }
 
+  /**
+   * Apply slant feature.
+   *
+   * @param slantControl slant value within range 0~1
+   */
+  public void applySlant(double slantControl) {
+    double slantRadian = slantControl * Math.PI / 2 * SLANT_MAX;
+    for (ArrayList<BezierCurve> contour : contours) {
+      for (int i = 0; i < contour.size(); i++) {
+        BezierCurve bezierCurve = contour.get(i);
+        Vector2D[] points = bezierCurve.getPoints();
+        for (int j = 0; j < points.length; j++) {
+          Vector2D point = points[j];
+
+          double newX = point.getX()
+              + (point.getY() - locatorRegion.getMinY()) / (Math.tan(Math.PI / 2 - slantRadian));
+
+          bezierCurve.setPoint(j, new Vector2D(newX, point.getY()));
```

Comment:
`setPoint()` in for loop is not efficient because each `setPoint()` call re-calculates the polynomials of the Bezier curve.

How about create `setPoints()` in `BezierCurve` and just using it after for loop? Below would be enough.

```java
/**
 * Sets points of the curve
 *
 * @param points new points
 * @throws OutOfRangeException if order is not same to the previous order of the curve
 */
public void setPoints(Vector2D[] points) throws OutOfRangeException {
  MathUtils.checkNotNull(points);
  final int order = points.length - 1;
  if (order != this.order) {
    throw new OutOfRangeException(order, this.order, this.order);
  }
  for (Vector2D point : points) {
    MathUtils.checkNotNull(point);
  }

  // Copy points.
  Vector2D[] copiedPoints = points.clone();
  for (int i = 0; i <= order; i++) {
    copiedPoints[i] = new Vector2D(1, copiedPoints[i]);
  }

  // Calculate coefficients.
  double[][] coefficients = calculateCoefficients(copiedPoints);

  // Save variables.
  this.points = copiedPoints;
  this.polynomials = new PolynomialFunction[2];
  this.polynomials[0] = new PolynomialFunction(coefficients[0]);
  this.polynomials[1] = new PolynomialFunction(coefficients[1]);
}
```